examples/supermassive-todomvc/package.json

```diff
@@ -3,7 +3,7 @@
   "license": "MIT",
   "main": "index.js",
   "private": true,
-  "version": "1.0.1-0",
+  "version": "1.0.1-1",
```

Comment:
I have manually released the new version in order to update license info. Since the original version did not have license. But this package should be private and there is not reason to ever publish this example.